arbitrator/prover/src/wavm.rs

```diff
@@ -67,14 +67,17 @@ pub enum IBinOpType {
     Rotr,
 }
 
+impl IBinOpType {
+    pub fn signed(&self) -> bool {
+        use IBinOpType::*;
+        matches!(self, DivS | RemS | ShrS)
```

Comment:
Is this function ever invoked? I didn't see an invocation but I might've missed it.

Reply:
Oops, that was useful earlier. Removed